src/dialogs/ColumnPicker.js

```diff
@@ -22,8 +22,11 @@ var ColumnPicker = Dialog.extend('ColumnPicker', {
         this.grid = grid;
 
         if (behavior.isColumnReorderable()) {
-            // grab the lists from the behavior
-            if (behavior.setGroups){
+            // parse & add the drag-and-drop stylesheet addendum
+            var stylesheetAddendum = stylesheet.inject('list-dragon-addendum');
+
+            // grab the group lists from the behavior
+            if (behavior.setGroups) {
```

Comment:
@joneit I think I need to hookup this method to your drop event (or rather close dialog). Also I can move this to the addons folder